.vib/gitlab-runner-helper/goss/vars.yaml

```diff
@@ -0,0 +1,10 @@
+binaries:
+  - gitlab-runner-helper
+  - gosu
+files:
+  - paths:
+      - /opt/bitnami/gitlab-runner-helper/bin/gitlab-runner-build
```

Comment:
I'm not sure where are we obtaining this path from; I do not see any mention of it on the asset's compilation or setup logic. Even so, given that it is a binary I think it would be better to move this check to the `binaries` section (if necessary).